src/components/note-modal/NoteModal.jsx

```diff
@@ -1,37 +1,99 @@
 import React, { useState } from "react";
+import ReactQuill from "react-quill";
+import "react-quill/dist/quill.snow.css";
+import { LabelModal } from "./label-modal/LabelModal";
```

Comment:
you can do a barrel import/export , make the code look clean

Reply:
they are just two files so I am not doing that .